src/constants/typography.ts

```diff
@@ -3,7 +3,7 @@ const typography = {
   sizes: {
     text: {
       lead: '18px',
-      body: '15px',
+      body: '16px',
```

Comment:
There's a voice in the back of my head saying we should do this as part of a major release due to the size change could do a lot of difference in how things look and it may be risky to upgrade if for instance a a button gets bigger and won't fit on the row it's on. But I'm happy if we can make this change now.

Reply:
I think that's fair now I look at it I just saw that @raff-r already has a PR open for the text-size increase maybe he's already done that testing?

Reply:
@JuliaHempel I think the text changes are going to take longer to get tested and approved so I've stripped them out of this PR so we can continue with the design QA.

Reply:
@moshie font size changes should be live early next week